app/services/recovery_code_generator.rb

```diff
@@ -1,6 +1,8 @@
 class RecoveryCodeGenerator
   attr_reader :user_access_key
 
+  INVALID_CODE = 'this is an invalid recovery code but a valid string'.freeze
```

Comment:
Ah I get it. The value didn't not seem super clear to me, got any other alternatives?